Formula/hledger.rb

```diff
@@ -28,18 +28,18 @@ class Hledger < Formula
   uses_from_macos "zlib"
 
   resource "hledger-lib" do
-    url "https://hackage.haskell.org/package/hledger-lib-1.22.2/hledger-lib-1.22.2.tar.gz"
-    sha256 "b4c54f11e7efdc0fcb01050300f02899c8a96dc88061821fc5984aecbb796877"
+    url "https://hackage.haskell.org/package/hledger-lib-1.3/hledger-lib-1.23.tar.gz"
```

Comment:
```suggestion
    url "https://hackage.haskell.org/package/hledger-lib-1.23/hledger-lib-1.23.tar.gz"
```
The checksum probably also needs fixing.

Reply:
Yes, I just updated the resources in the 2nd commit.